spec/factories/buses.rb

```diff
@@ -0,0 +1,9 @@
+require "faker"
```

Comment:
Please check if we can skip this line

Reply:
Sorry apu, I didn't find any solution where `Faker` gem worked without the `require "faker"`. Is there any possible way to skip that line, that i'm missing?

Reply:
Shifted the faker to rails_helper.rb file and skip it from this file. Thanks!